controllers/api/familyController.js

```diff
@@ -136,10 +157,41 @@ async function updateFamilyMember (req, res) {
     }
 }
 
+async function patchFamilyMember(req, res) {
+  const id = req.params.id;
+  if (!id) {
+    return res.json({ error: "ID is required" });
+  }
+
+  const user = await Family.findById(id);
+
+  if (!user) {
+    return res.json({ error: `No user with an ID of ${id} could be found` });
+  }
+
+  Object.entries(req.body).forEach(([key, value]) => {
+    user[key] = value;
+  });
+
+  await user.save()
```

Comment:
When working with an Object Relational Mapper (ORM) such as Mongoose, Active Record, Sequelize... There is often a `save` method associated with an instance of a model. This can be used to update the database record associated with the instantiated model instance.

Reply:
That's fantastic. Why the hell weren't we taught this. Instead I've been having to clear the database after rewriting the model and then reenter the data.